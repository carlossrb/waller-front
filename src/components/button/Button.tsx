import React, { FC } from 'react';

export const Button: FC = ({ children }) => {
  return <button className="bg-primary400 text-danger50">{children}</button>;
};
