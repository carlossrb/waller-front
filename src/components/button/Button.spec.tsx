import { render, fireEvent } from '@testing-library/react';
import React from 'react';
import { Button } from './Button';

describe('Button', () => {
  it('should render button', () => {
    const { getByText } = render(<Button>Hello</Button>);
    expect(getByText('Hello')).toMatchSnapshot();
  });
});
