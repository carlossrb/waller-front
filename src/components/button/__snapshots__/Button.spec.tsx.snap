// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should render button 1`] = `
<button
  class="bg-primary400 text-danger50"
>
  Hello
</button>
`;
